bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -62,3 +62,10 @@ spec:
       for: 15m
       labels:
         severity: warning
+    - alert: OVNKubernetesResourceRetryFailure
+      annotations:
+        summary: OVN Kubernetes failed numerous times to apply networking control plane configuration.
+        description: OVN Kubernetes failed to apply networking control plane configuration after attempting numerous times. Network control plane configuration may not be applied.
+      expr: changes(ovnkube_resource_retry_failures_total[15m]) > 0
```

Comment:
Does this mean that if a resource failed to be applied, a cluster admin would see this alert firing for 15 mins?

Reply:
Yes @martinkennelly , The alert was seen for 15 mins before it got disappeared.
![Screenshot from 2023-02-20 17-08-11](https://user-images.githubusercontent.com/30619230/220327470-ca673c81-1a45-4d60-8b7f-69069e624d27.png)